src/Highway.Data.EntityFramework/Interfaces/IEntityDataContext.cs

```diff
@@ -1,22 +1,11 @@
-﻿using System.Collections.Generic;
-using System.Data.Common;
+﻿using System.Data.Common;
 
 namespace Highway.Data.EntityFramework
 {
     /// <summary>
     /// </summary>
-    public interface IEntityDataContext : IDataContext
+    public interface IEntityDataContext : IDataContext, IReadonlyEntityDataContext
     {
-        /// <summary>
```

Comment:
It could be argued that moving this method to the read-only interface isn't safe, if it is just a SQL string and params. But the intent of the name "ExecuteSqlQuery" seems obvious. Thoughts?